src/tabs/tabs.js

```diff
@@ -132,6 +133,10 @@ angular.module('ui.bootstrap.tabs', [])
         });
       }
 
+      if (attrs.classes === undefined) {
```

Comment:
Angular has built in functions for checking if variables are [undefined](https://docs.angularjs.org/api/ng/function/angular.isUndefined) or are [defined](https://docs.angularjs.org/api/ng/function/angular.isDefined)